### DISCUSSION TITLE
osu lazer download error

### DISCUSSION THREAD

**Discussion by cookieOHO** [ppy/osu#20831] [Q&A]
### Type

Other

### Bug description

How to fix this window when trying to download?
Use google translate, sorry if wrong

### Screenshots or videos

![image](https://user-images.githubusercontent.com/116189747/196738284-526b91b2-0f74-4e4e-950c-925f2c223c6a.png)

### Version

2022.1008.2

### Logs

[Squirrel.log](https://github.com/ppy/osu/files/9821986/Squirrel.log)

**Comment by bdach** [comment]
There's nothing of interest in the squirrel log attached. Error code returned suggests generic access violation if I didn't convert wrong (0xc000005).

Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by frenzibyte** [reply]
Interestingly, converting the shown error code on a quick C# instance shows it to be 0xC0000006, not sure if that's a display issue from Squirrel or it's actually signifying another error entirely (which, briefly checking on google, could be a "Bad Image" error that may occur when running the installer on a network drive or otherwise).

**Reply by bdach** [reply]
I thought it may be ...06. Off by one when converting two's complement. Doesn't help anything either way given the lack of specificity.

**Reply by cookieOHO** [reply]
[Event viewer.zip](https://github.com/ppy/osu/files/9831701/WinRAR.ZIP.zip)

**Reply by bdach** [reply]
Not much more in the event viewer log. There's this:

> Windows cannot access the file  for one of the following reasons: there is a problem with the network connection, the disk that the file is stored on, or the storage drivers installed on this computer; or the disk is missing. Windows closed the program Update because of this error.
>
> Program: Update
> File: 
>
> The error value is listed in the Additional Data section.
> User Action
> 1. Open the file again. This situation might be a temporary problem that corrects itself when the program runs again.
> 2. If the file still cannot be accessed and
>	- It is on the network, your network administrator should verify that there is not a problem with the network and that the server can be contacted.
>	- It is on a removable disk, for example, a floppy disk or CD-ROM, verify that the disk is fully inserted into the computer.
> 3. Check and repair the file system by running CHKDSK. To run CHKDSK, click Start, click Run, type CMD, and then click OK. At the command prompt, type CHKDSK /F, and then press ENTER.
> 4. If the problem persists, restore the file from a backup copy.
> 5. Determine whether other files on the same disk can be opened. If not, the disk might be damaged. If it is a hard disk, contact your administrator or computer hardware vendor for further assistance.

Does this message help anything?
